javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -177,9 +186,9 @@ public void tearDown() {
 
         @Benchmark
         public Object java_mutable(Initialized state) {
-            final java.util.ArrayList<Integer> values = state.javaMutable;
+            java.util.List<Integer> values = state.javaMutable;
             for (int i = 0; i < CONTAINER_SIZE; i++) {
-                values.remove(0);
+                values = values.subList(1, values.size()); // remove(0) would copy everything, but this will slow access down because of nesting
```

Comment:
`remove(0)` might be cheating (i.e. making `ArrayList` look worst), but `subList` has its drawbacks also, i.e. access will depend on the number of `subList` wrappers